### PR TITLE
feat: avatar comp added

### PR DESCRIPTION
**Pull request by nyyshaaa**
avatar-badge

### CODE REVIEW

**Review by shashikantmaurya0007 — APPROVED** [review]
Looks good to me .you can merge your code :)

File: pages/components/Avatar-badge/avatar-badge.css [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    width: 1.3rem;
+    bottom: 1.3rem;
+    right: 0.4rem;
+    border-radius:50% ;
+    border: 2px solid var(--blue-purple);
+
+}
+
+.online{
+    background-color:#00edc6;
```
Comment (AtulHande03a) [inline pages/components/Avatar-badge/avatar-badge.css:57]: use variables for colors

File: pages/components/Avatar-badge/avatar-badge.css [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+}
+
+.offline{
+    background-color: #8089c7;
```
Comment (AtulHande03a) [inline pages/components/Avatar-badge/avatar-badge.css:61]: use variables for colors

File: pages/components/Avatar-badge/avatar-badge.css (hunk 1 above)
Comment (nyyshaaa) [reply pages/components/Avatar-badge/avatar-badge.css:57]: yeah i forgot to do.will keep in mind from next time.Thank you !

**Comment by AtulHande03a** [comment]
naming conventions looks good. Use variables when using colors. Rest looks good.

**Pull request merged**